docs/Migrating to Newer SDL Versions/Migrating SDL Core 6.1 to 7.0/index.md

```diff
@@ -0,0 +1,276 @@
+
+## Breaking Changes
+
+- The `url` parameter in the HMI API and RPC Spec has removed its max length requirement.
+
+- The parameter `appID` has been removed from VehicleInfo.UnsubscribeVehicleData request to the HMI.
+
+## Newly Deprecated
+
+### Deprecated Character Sets
+
+Character sets `TYPE2SET`, `TYPE5SET`, `CID1SET`, and `CID2SET` have been deprecated. These character sets only had proprietary significance and HMI's can now choose from the following character sets:
+
+- ASCII
+- ISO_8859_1
+- UTF_8
+
+### Deprecated HMI RPC: OnFindApplications
+
+This unimplemented RPC has been marked as deprecated and should be removed in the next major version change of SDL Core. 
+
+### Deprecated Vehicle Data
+
+Vehicle Data parameters `fuelLevel` and `fuelLevel_state` have been deprecated. Please make updates to use expanded vehicle data struct `FuelRange`.
+
+```
+<struct name="FuelRange">
+    <param name="type" type="Common.FuelType" mandatory="false"/>
+    <param name="range" type="Float" minvalue="0" maxvalue="10000" mandatory="false">
+        <description>
+            The estimate range in KM the vehicle can travel based on fuel level and consumption.
+        </description>
+    </param>
+    <param name="level" type="Float" minvalue="-6" maxvalue="1000000" mandatory="false">
+    <description>The relative remaining capacity of this fuel type (percentage).</description>
+    </param>
+    <param name="levelState" type="Common.ComponentVolumeStatus" mandatory="false">
+        <description>The fuel level state</description>
+    </param>
+    <param name="capacity" type="Float" minvalue="0" maxvalue="1000000" mandatory="false">
+    <description>The absolute capacity of this fuel type.</description>
+    </param>
+    <param name="capacityUnit" type="Common.CapacityUnit" mandatory="false">
+    <description>The unit of the capacity of this fuel type such as liters for gasoline or kWh for batteries.</description>
+    </param>
+</struct>
+```
+
+Vehicle Data parameter `prndl` has been deprecated. Please make updates to use the new vehicle data struct `GearStatus`.
+
+```
+<struct name="GearStatus">
+    <param name="userSelectedGear" type="Common.PRNDL" mandatory="false">
+        <description>Gear position selected by the user i.e. Park, Drive, Reverse</description>
+    </param>
+    <param name="actualGear" type="Common.PRNDL" mandatory="false">
+        <description>Actual Gear in use by the transmission</description>
+    </param>
+    <param name="transmissionType" type="Common.TransmissionType" mandatory="false">
+        <description>Tells the transmission type</description>
+    </param>
+</struct>
+```
+
+## Additions
+
+### Vehicle Data
+
+- FuelRange was expanded to replace `fuelLevel` and `fuelLevel_state` parameters
+- New vehicle data type: GearStatus to replace `prndl` parameter
+- New vehicle data type: StabilityControlStatus
+- New vehicle data type: WindowStatus
+- New vehicle data type: HandsOffSteering
```

Comment:
Should code blocks be added around the new vehicle data?

Reply:
added